### DISCUSSION TITLE
error on windows 11

### DISCUSSION THREAD

**Discussion by promedacademy** [ParisNeo/Gpt4All-webui#25] [General]
i did all steps however on launching the app I get this error:
Virtual environment created and packages installed successfully.
Launching application...
Traceback (most recent call last):
  File "C:\gpt\gpt4web\Gpt4All-webui\app.py", line 44, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'

**Comment by ParisNeo** [comment]
Already fixed in last version
